api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -15,7 +15,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Code generated by controller-gen. DO NOT EDIT.
+// autogenerated by controller-gen object, do not modify manually
```

Comment:
Can you check the version of `controller-gen` on your local machine. Looks like you might be using a different one (probably older).

```
$ controller-gen --version
Version: v0.2.4
```

If you use the correct version as above, these spurious changes will go away.

Reply:
yeah, you're right... had been investigating newer and older kubebuilder versions with Ravi recently re: another issue and I think I ended up building and linking an older version of controller-tools source
 
not sure what version I was on... it was before they even added the `--version` flag!

anyway updated binary and then quashed some incorrectly generated code/spec in latest commit